### ISSUE TITLE
More detailed wiki

### ISSUE THREAD

**Issue by Boddebon** [funniray/minimap-control#6]
Hello, I would like more detailed documentation on the plugin, from setting up each item in the config, to commands and permissions

**Comment by funniray** [comment]
Hey there.

None of the config options are really documented for a couple of reasons.
1. Configurate doesn't allow for comments in configuration files
2. I don't actually control what most of the options actually do. The behavior is directly controlled by the mods themselves
3. If you want to configure Journeymap, use the in game "server admin" ui. It works extremely well with the plugin and explains the config options

There are no commands currently

The only permission node is `minimap.jm.admin`, which is required for the journeymap "server admin" ui.

**Comment by Boddebon** [comment]
Sorry, did I understand you correctly, if I give a player permission minimap.jm.admin, then I can go into the admin window through the minimap interface and edit the settings for all players with the map there?

**Comment by Boddebon** [comment]
Is it possible to do this in Xaero's Minimap, because I don't see an admin panel there. As I understand it, you need to use the exact mod that you use to edit via the admin panel? And what mods does the plugin work with in general at the time of its latest version?

**Comment by funniray** [comment]
> Is it possible to do this in Xaero's Minimap
No. There's about 3 config options (optionally per world) for Xaero's and the mod itself doesn't support it. If the mod supported it, I would add it.

> you need to use the exact mod that you use to edit via the admin panel
As far as I'm aware, they haven't changed the options in the menu since I've started. You should be able to use any modern (1.21 or 1.20) version of the mod. If the client is missing an option, then it'll silently be set to the default.

> And what mods does the plugin work with
Journeymap
XaerosMinimap
VoxelMap-Updated

**Comment by zefir-git** [comment]
Just to clarify, to configure the JourneyMap options, do I need to install a client with JourneyMap and use a server admin configuration GUI?
Do you know if there is any documentation by JourneyMap instead?

**Comment by funniray** [comment]
The client mod has a built in UI that documents everything each option does. As far as I'm aware, I don't believe journeymap has documentation anywhere else on what the options do.
